etc/helm/pachyderm/templates/pachd/identity-config.yaml

```diff
@@ -30,7 +30,7 @@ data:
       redirect_uris:
       - {{ include "pachyderm.consoleRedirectURI" . | quote }}
     {{- end }}
-    {{- if .Values.determined.enabled }}
```

Comment:
I believe that this should also still block on determined.enabled, in addition to the other values

Reply:
Updated!